lib/tasks/data.rake

```diff
@@ -32,5 +32,14 @@ namespace :data do
       puts "Transaction #{transaction.id} with cc# "\
            "#{transaction.credit_card_number} created."
     end
+
+    CSV.foreach('./db/data/items.csv', headers: true) do |row|
+      item = Item.create(name: row['name'],
```

Comment:
when seeding data or in your tests, it's a good idea to use the bang methods - so using `create!` instead of `create`  - this will cause ActiveRecord to fail loudly and let you know if an `Item` is not created. as it is now, if a specific row doesn't get translated to an `Item`, you won't actually know about it!